hooks/validator/README.md

```diff
@@ -0,0 +1,86 @@
+# Validator Hook
+
+![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
+![Alpha](https://img.shields.io/badge/alpha-release-red)
+
+The `validator hook` taps into the `After` lifecycle event to validate the result of flag evaluations.
+
+The hook defines a `validator` interface with function
+```go
+IsValid(flagEvaluationDetails of.EvaluationDetails) error
+```
+to allow application authors to supply their own validators.
+
+There are, however, [ready to be used validators](#validators) that conform to the interface.
+
+## Setup
+
+Import the [OpenFeature SDK](https://github.com/open-feature/golang-sdk) and the validator.
+
+Create an instance of the validator hook struct (using the hex validator as an example):
+```go
+package main
+
+import (
+"github.com/open-feature/golang-sdk-contrib/hooks/validator/pkg/hex"
+"github.com/open-feature/golang-sdk-contrib/hooks/validator/pkg/validator"
+"github.com/open-feature/golang-sdk/pkg/openfeature"
+)
+
+func main() {
+	v := validator.Hook{Validator: hex.Validator{}}
+}
+```
+
+Use the validator hook (on invocation as an example):
+```go
+client := openfeature.NewClient("foo")
+evalOptions := openfeature.NewEvaluationOptions([]openfeature.Hook{v}, openfeature.HookHints{})
+value, err := client.
```

Comment:
Can you also confirm that value contains the fallback string `#0000FF`?

Reply:
I'm assuming you mean in the case where the resolved flag is invalid?
Otherwise that's the default behaviour